helm/stargate/templates/stargate_authapi_service.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
```

Comment:
Few general things:

* can we add a small comment at top of the each yaml file, describing the service/deployment, etc it can help others to understand everything and connect dots
* yaml files should start with `---`, or?

Reply:
Instead of adding comment to the yaml file, I will add a readme file.
Not sure why we need ---, if I have multiple resource configuration in the same yaml file i have seen using --- to separate them.